pkg/api/validation/dynakube/oneagent.go

```diff
@@ -191,3 +197,53 @@ func isOneAgentVersionValid(_ context.Context, _ *Validator, dk *dynakube.DynaKu
 
 	return ""
 }
+
+func hasNoDuplicatOneAgentArguments(_ context.Context, _ *Validator, dk *dynakube.DynaKube) string {
```

Comment:
```suggestion
func duplicateOneAgentArguments(_ context.Context, _ *Validator, dk *dynakube.DynaKube) string {
```

I tried to follow the naming of other functions

Reply:
Fixed in [5d093cf](https://github.com/Dynatrace/dynatrace-operator/pull/4377/commits/5d093cf3fe7092cda67d5f0f73208e02a81e2716)